src/midx.c

```diff
@@ -399,6 +441,23 @@ int git_midx_entry_find(
 	return 0;
 }
 
+int git_midx_foreach_entry(
+		git_midx_file *idx,
+		git_odb_foreach_cb cb,
+		void *data)
+{
+	size_t i;
+	int error;
+
+	assert(idx);
+
+	for (i = 0; i < idx->num_objects; ++i)
```

Comment:
Stylistically, I think that we prefer braces around branching statements, treating an `if` as an implicit multi-line statement.  (Not sure if I'm describing this well.)  eg:

```
for (i = 0; i < idx->num_objects; ++i) {
    if ((error = cb(&idx->oid_lookup[i], data)) != 0)
        return git_error_set_after_callback(error);
}
```

I say "I think" because there are a handful of stylistic grey areas, and I think that this is one of them.  Maybe this should be better expressed as "it's my opinion that ...".

No need to change this - again, grey area, perfectly reasonable - I just thought that I'd mention it.

Reply:
Let's change it, I needed to push a new change anyways.